src/components/chip/chip.tsx

```diff
@@ -166,6 +175,14 @@ export class Chip {
         );
     }
 
+    private renderBadge() {
+        if (!this.badge) {
+            return;
+        }
```

Comment:
It seems like a feature, rather than a bug, but just to make sure:

It's intentional (or a happy coincidence 😄) that `badge={0}` renders no badge, I assume?

Reply:
I don't remember really. But some faint memory tells me that it was intentional. :)